signer/src/storage/postgres.rs

```diff
@@ -111,7 +114,13 @@ impl TryFrom<&NakamotoBlock> for model::StacksBlock {
 impl PgStore {
     /// Connect to the Postgres database at `url`.
     pub async fn connect(url: &str) -> Result<Self, Error> {
-        let pool = sqlx::PgPool::connect(url)
+        let pool = PgPoolOptions::new()
```

Comment:
Doing this lets us remove schema qualifiers from the queries in this file, which makes them easier to read.